### PR TITLE
Initial commit.

### PR DESCRIPTION
**Pull request by michdud**
(@willlarche it won't let me add you as a reviewer, but if you could take a look, please!)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,21 @@
 # material-components-flutter-experimental
-Work in progress code, implementation options, and design prototypes. Organized by branch.
+This repository contains work in progress code, implementation options, and design prototypes for  [Material Components for Flutter](https://github.com/material-components/material-components-flutter). Organized by branch.
```
Comment (willlarche) [inline README.md]: Extra space before link I think

File: AUTHORS [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# This is the list of Material Components for Flutter Codelabs authors for
+# copyright purposes.
+#
+# This does not necessarily list everyone who has contributed code, since in
+# some cases, their employer may be the copyright holder.  To see the full list
+# of contributors, see the revision history in source control.
+Google Inc.
```
Comment (willlarche) [inline AUTHORS]: Add new line at end.

File: README.md (hunk 1 above)
Comment (michdud) [reply README.md]: Removed extra space

File: AUTHORS (hunk 2 above)
Comment (michdud) [reply AUTHORS]: Done!

File: README.md [hunk 3]
```diff
@@ -1,2 +1,21 @@
 # material-components-flutter-experimental
-Work in progress code, implementation options, and design prototypes. Organized by branch.
+This repository contains work in progress code, implementation options, and design prototypes for [Material Components for Flutter](https://github.com/material-components/material-components-flutter). Organized by branch.
+
+## License
```
Comment (willlarche) [inline README.md]: Cut the license section from the README

Comment (michdud) [reply README.md]: Done!

**Comment by willlarche** [comment]
Go ahead and merge!

**Pull request merged**